src/content/docs/opentelemetry/best-practices/opentelemetry-otlp.mdx

```diff
@@ -213,12 +214,12 @@ See [metric attribute limits](/docs/data-apis/ingest-apis/metric-api/metric-api-
 
 The mechanism to configure attribute limits will vary, but OpenTelemetry language SDKs generally support setting the `OTEL_ATTRIBUTE_VALUE_LENGTH_LIMIT=4095` and `OTEL_ATTRIBUTE_COUNT_LIMIT=64` environment variables (see [OpenTelemetry docs](https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/) for more info).
 
-If using the collector, the [transform processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/transformprocessor) can be configured to truncate attributes to New Relic limits.
+If using the Collector, the [transform processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/transformprocessor) can be configured to truncate attributes to New Relic limits.
```

Comment:
I would write this to avoid the passive voice:
`When using the collector, you can configure the [transform processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/transformprocessor) to truncate attributes to New Relic limits.`